### PR TITLE
Extract Factory Bot cops

### PR DESCRIPTION
**Pull request by ydah**

### CODE REVIEW

**Comment by ydah** [comment]
The following steps will be used for the final extraction:
https://github.com/rubocop-hq/rubocop-capybara/pull/1#issuecomment-1363270769

File: .rubocop.yml [hunk 1]
```diff
@@ -143,8 +143,6 @@ RSpec/NoExpectationExample:
   Enabled: true
 RSpec/PendingWithoutReason:
   Enabled: true
-RSpec/RedundantAround:
-  Enabled: true
```
Comment (pirj) [inline .rubocop.yml:147]: 👍 Not released yet.

File: lib/rubocop/factory_bot/version.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 module RuboCop
-  module RSpec
-    # Version information for the RSpec RuboCop plugin.
+  module FactoryBot
+    # Version information for the FactoryBot RuboCop plugin.
     module Version
       STRING = '2.18.1'
```
Comment (pirj) [inline lib/rubocop/factory_bot/version.rb]: Maybe start with 2.19.0, as it would correspond to the version of `rubocop-rspec` where it will be extracted?

**Review by pirj — APPROVED** [review]
Looks great, thank you, @ydah !

**Comment by pirj** [comment]
What might be missing is the changelog. I suggest only keeping what's related to FB cops.

File: lib/rubocop/factory_bot/version.rb (hunk 2 above)
Comment (pirj) [reply lib/rubocop/factory_bot/version.rb]: Or 2.20.0 if we decide to release what we currently have unreleased in 2.19.0. WDYT @bquorning ?

Comment (ydah) [reply lib/rubocop/factory_bot/version.rb]: I thought it looked better to use 2.20.0, so I changed it

**Comment by ydah** [comment]
@pirj I updated this PR. Thank you for your review!

File: rubocop-factory_bot.gemspec [hunk 3]
```diff
@@ -1,25 +1,20 @@
...
+  spec.name = 'rubocop-factory_bot'
+  spec.summary = 'Code style checking for FactoryBot files'
   spec.description = <<-DESCRIPTION
-    Code style checking for RSpec files.
+    Code style checking for FactoryBot files.
     A plugin for the RuboCop code style enforcing & linting tool.
   DESCRIPTION
-  spec.homepage = 'https://github.com/rubocop/rubocop-rspec'
+  spec.homepage = 'https://github.com/rubocop/rubocop-factory_bot'
   spec.authors = ['John Backus', 'Ian MacLeod', 'Nils Gemeinhardt']
```
Comment (ydah) [inline rubocop-factory_bot.gemspec:14]: memo) Who would be a suitable author?

**Review by pirj — APPROVED** [review]
Looks flawless, thank you! 🙌

**Comment by ydah** [comment]
The following changes to the FactoryBot department have been incorporated
- https://github.com/rubocop/rubocop-rspec/pull/1601

**Pull request closed without merging**